### DISCUSSION TITLE
How to ignore failed updates in the gui?

### DISCUSSION THREAD

**Discussion by UsefulVid** [Devolutions/UniGetUI#2305] [Q&A]
I like this project but my user experience lacks on some ocations.
I tried to update all of my software and there are some error messages. (I am using winget as packet manager)
![image](https://github.com/marticliment/WingetUI/assets/39967070/95eaade1-f274-484d-9d72-4deabcfac4a4)
No problem with that so far.
No comes the question: how can I ignore them for the next update?
If I click on "installed packages" the list only shows winget, nothing I could select here
![image](https://github.com/marticliment/WingetUI/assets/39967070/91183634-3a78-4e3d-9138-5d6b31abfff6)
I cannot find any list there I can easily select the packages which I like to ignore.

**Comment by marticliment** [comment]
Go to Software Updates -> Right-click the faulting package(s) -> Skip this version or Ignore Updates

**Reply by UsefulVid** [reply]
This list was empty. I just restarted and after UniGetUI tried to update again, the packages reappeared.

**Reply by marticliment** [reply]
Which version of WingetUI/UniGetUI are you using?

**Reply by UsefulVid** [reply]
3.1.0 alpha1

**Reply by marticliment** [reply]
Please attach your logs here

**Reply by UsefulVid** [reply]
[UniGetUI-Log2.txt](https://github.com/user-attachments/files/15815999/UniGetUI-Log2.txt)
[UniGetUI-Log.txt](https://github.com/user-attachments/files/15816000/UniGetUI-Log.txt)

**Reply by marticliment** [reply]
I will see what I can do.

Meanwhile, please either downgrade to 3.0.2 or wait for the next beta release
